### PR TITLE
Seabeta 333 update heating pattern

### PR DESCRIPTION
**Pull request by CodrinCotarlanSoftwire**

### CODE REVIEW

File: SeaPublicWebsite/Services/RecommendationService.cs [hunk 1]
```diff
@@ -461,8 +461,6 @@ private static BreHeatingPatternType GetBreHeatingPatternType(HeatingPattern? he
             {
                 HeatingPattern.AllDayAndNight => BreHeatingPatternType.AllDayAndAllNight,
                 HeatingPattern.AllDayNotNight => BreHeatingPatternType.AllDayButOffAtNight,
-                HeatingPattern.MorningAndEvening => BreHeatingPatternType.MorningAndEvening,
-                HeatingPattern.OnceADay => BreHeatingPatternType.JustOnceADay,
```
Comment (CodrinCotarlanSoftwire) [inline SeaPublicWebsite/Services/RecommendationService.cs:465]: Since we removed those options, we won't need these anymore.

**Comment by DanCorderSoftwire** [comment]
Am I missing something, or are we not sending those numbers to BRE at all?

**Comment by CodrinCotarlanSoftwire** [comment]
> Am I missing something, or are we not sending those numbers to BRE at all?

They are. In `CreateRequest`
...
`BreHeatingPatternType breHeatingPatternType = GetBreHeatingPatternType(userData.HeatingPattern.Value);

            BreRequest request = new(
                brePostcode: userData.Postcode,
                brePropertyType: brePropertyType,
                breBuiltForm: breBuiltForm,
                breFlatLevel: breFlatLevel,
                breConstructionDate: breConstructionDate,
                breWallType: breWallType,
                breRoofType: breRoofType,
                breGlazingType: breGlazingType,
                breHeatingFuel: breHeatingFuel,
                breHotWaterCylinder: breHotWaterCylinder,
                breOccupants: userData.NumberOfOccupants,
                breHeatingPatternType: breHeatingPatternType,
                breTemperature: userData.Temperature
            );

            return request;`

**Comment by DanCorderSoftwire** [comment]
That's heating pattern, not the two numbers for the number of hours of heating.

**Comment by CodrinCotarlanSoftwire** [comment]
Ah yes you're right! It may be the case the BRE API does not take into account custom heating hours? (It didn't use the previous hours either)...

**Comment by DanCorderSoftwire** [comment]
Ok, sounds like a separate BRE integration issue

**Review by DanCorderSoftwire — APPROVED** [review]

**Pull request merged**